UI/client/signup.js

```diff
@@ -24,25 +23,26 @@ const signup = (e) => {
       body: JSON.stringify({ name, email, password, address, phone }),
 
   })
-    .then(response => response)
+    .then(response => response.json())
     .then((result) => {
         console.log(result);
-        console.log(result.headers.get('x-auth-token'));
-      if (result.status !== 201) {
+        console.log(result.token);
+      if (result.state !== 'Succesful') {
         showNotification.style.display = 'block';
         showNotification.style.background = 'red';
-        showNotification.innerHTML = 'Please input correct values for all fields';
+        showNotification.innerHTML = result.message;
         setInterval(() => {
           showNotification.style.display = 'none';
         }, 2000);
       } else {
-        localStorage.token = result.headers.get('x-auth-token');
+        localStorage.token = result.token;
          console.log(localStorage.token + 'YAAAAY');
```

Comment:
#### Critical
Why do we have `console.log()` here?
You should remove all instances of logging in your code when you are going live.